inference/huggingface/zero_inference/run_model.py

```diff
@@ -155,32 +154,25 @@ def run_generation(
     quant_group_size,
     pin_kv_cache,
     async_kv_offload,
+    loops,
 ):
     # Load tokenizer
-    config = get_model_config(model_name)
-    return_token_type_ids = True 
-    padding_side = "left" if config.model_type in ["opt"] else "right"
+    config = get_model_config(model_name)    
 
     if config.model_type == "opt":
         tokenizer = AutoTokenizer.from_pretrained(
             model_name.replace("175b", "66b"), 
```

Comment:
Why do we change the model name from 175b to 66b? Is this a hack because OPT model does not have the 175b config? If so, it might be helpful to add a comment to explain the reason.

Reply:
Great suggestion. Done!